modules/luxon/src/impl/diff.js

```diff
@@ -0,0 +1,76 @@
+import Duration from '../duration.js';
+
+function dayDiff(earlier, later) {
+  const utcDayStart = (dt) => dt.toUTC(0, { keepLocalTime: true }).startOf('day').valueOf();
+  const ms = utcDayStart(later) - utcDayStart(earlier);
+  return Math.floor(Duration.fromMillis(ms).as('days'));
+}
+
+function highOrderDiffs(cursor, later, units) {
+  const differs = [
+    ['years', (a, b) => b.year - a.year],
+    ['quarters', (a, b) => b.quarter - a.quarter],
+    ['months', (a, b) => b.month - a.month + (b.year - a.year) * 12],
+    [
```

Comment:
- [ ] Great job for the function implementation, :clap: but I think it will be better if you can use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)